spec/index.bs

```diff
@@ -5832,6 +5832,7 @@ various encodings allowed by {{GPUQueue/copyExternalImageToTexture()}}.
 <script type=idl>
 dictionary GPUImageCopyExternalImage {
     required (ImageBitmap or HTMLCanvasElement or OffscreenCanvas) source;
+    boolean originBottomLeft = false;
```

Comment:
I haven't followed all the technical detail on why it's needed, but if it *is* needed, why are we not making it a proper `enum` (even if with only 2 variants)? If I see `originBottomLeft = true`, it's non-ambiguous. But seeing `originBottomLeft = false` may be.

Reply:
This is closed now, but for future reference in case we revisit this:

I considered making this an enum but there are really only two options (the GL way, and the normal way). Other places (webgl, imagebitmap) call this "flip y", which is something you do in an operation, not something you tag an image with. (Internally in Chromium we also have topleft/bottomleft origins.) Maybe "flip y" would make the most sense though since this is really part of the operation and not the tagging. (If it were tagging it would be in GPUImageCopyTextureTagged, not GPUImageCopyExternalImage.)

Initially I was going to make it `origin: "top-left"` but we already have a field called `origin`.